crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -67,26 +67,19 @@ impl TextureAtlas {
     }
 
     /// Generate a `TextureAtlas` by splitting a texture into a grid where each
```

Comment:
This doc comment has definitely turned into a proper run in sentence 😄 

We should split the doc comment into the first line summary, then a new line, then a list describing each the non-obvious parameters.

Reply:
This is almost perfect. Just needs an extra new line after the first sentence so the IDEs handle it properly.